frontend/.prettierrc

```diff
@@ -0,0 +1,10 @@
+{
+  "semi": true,
+  "singleQuote": true,
+  "jsxSingleQuote": true,
```

Comment:
it is better to keep double quote for jsx specifically

Reply:
Fixed. Please check it out. Thanks.